integration-tests/browser.spec-int.js

```diff
@@ -142,6 +142,121 @@ test.describe('localization', () => {
     })
 })
 
+test.describe('fire button', () => {
+    test('by default no fire button is displayed', async ({ page }) => {
+        const mock = new MockData({ url: 'https://example.com' })
+        const messages = mockToExtensionDashboardMessage(mock)
+        const dash = await DashboardPage.browser(page, messages)
+        await dash.showsPrimaryScreen()
+        const buttons = await dash.fireButton().all()
+        expect(buttons).toHaveLength(0)
+    })
+
+    test('adding firebutton option to dashboard message shows fire button', async ({ page }) => {
+        const mock = new MockData({ url: 'https://example.com' })
+        const messages = mockToExtensionDashboardMessage(mock, true)
+        const dash = await DashboardPage.browser(page, messages)
+        await dash.fireButton().waitFor()
+        expect(await dash.fireButton().isVisible()).toBeTruthy()
```

Comment:
can you add a screenshot for this state (see the bottom of this file). Even if we just add one for the primary screen when the fire button is showing that would be good and prevent others from breaking it :)